### PR TITLE
Azure Kubernetes Service controls

### PR DESCRIPTION
**Pull request by sapsh-msft**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK/Framework/Core/SVT/Services/KubernetesService.ps1 [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Set-StrictMode -Version Latest 
+class KubernetesService: SVTBase
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Core/SVT/Services/KubernetesService.ps1]: Is entry added in SVTMapping table for KubernetesService?

File: src/AzSK/Framework/Configurations/SVT/Services/KubernetesService.json [hunk 2]
```diff
@@ -0,0 +1,172 @@
+{
+  "FeatureName": "KubernetesService",
+  "Reference": "aka.ms/azsktcp/KubernetesService",
+  "IsMaintenanceMode": false,
+  "Controls": [
+    {
+      "ControlID": "Azure_KubernetesService_Deploy_Enable_Cluster_RBAC",
+      "Description": "Cluster RBAC must be enabled in Kubernetes Service",
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Configurations/SVT/Services/KubernetesService.json]: Get controls reviewed by @sbyna-msft and @mprabhu11

Comment (sapsh-msft) [reply src/AzSK/Framework/Configurations/SVT/Services/KubernetesService.json]: Already sent for review. Once I get the feedback, I will incorporate in the next PR.

File: src/AzSK/Framework/Core/SVT/Services/KubernetesService.ps1 [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    { 
+		$this.GetResourceObject();
+    }
+
+	hidden [PSObject] GetResourceObject()
+    {
+        if (-not $this.ResourceObject) 
+		{
+			$ResourceAppIdURI = [WebRequestHelper]::AzureManagementUri;
+            $AccessToken = [Helpers]::GetAccessToken($ResourceAppIdURI)
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Core/SVT/Services/KubernetesService.ps1]: Use WebRequestHelper helper method to get management Uri

File: src/AzSK/Framework/Core/SVT/Services/KubernetesService.ps1 [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+			if($null -ne $AccessToken)
+			{
+
+				$header = "Bearer " + $AccessToken
+				$headers = @{"Authorization"=$header;"Content-Type"="application/json";}
+
+				$uri=[system.string]::Format("https://management.azure.com/subscriptions/{0}/resourceGroups/{1}/providers/Microsoft.ContainerService/managedClusters/{2}?api-version=2018-03-31",$this.SubscriptionContext.SubscriptionId, $this.ResourceContext.ResourceGroupName, $this.ResourceContext.ResourceName)
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Core/SVT/Services/KubernetesService.ps1]: Use WebRequestHelper helper method to get management Uri

File: src/AzSK/Framework/Core/SVT/Services/KubernetesService.ps1 [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+				$controlResult.VerificationResult = [VerificationResult]::Failed
+			}
+		}
+
+		return $controlResult;
+	}
+
+	hidden [controlresult[]] CheckKubernetesVersion([controlresult] $controlresult)
+	{
+		if(([Helpers]::CheckMember($this.ResourceObject,"Properties")) -and [Helpers]::CheckMember($this.ResourceObject.Properties,"kubernetesVersion"))
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Core/SVT/Services/KubernetesService.ps1]: Below single CheckMember should work 
([Helpers]::CheckMember($this.ResourceObject,"Properties.kubernetesVersion"))

**Pull request merged**